server/memphis_helper.go

```diff
@@ -519,6 +522,51 @@ func tryCreateInternalJetStreamResources(s *Server, retentionDur time.Duration,
 		INTEGRATIONS_AUDIT_LOGS_STREAM_CREATED = true
 	}
 
+	// create notifications stream
+	if !NOTIFICATIONS_BUFFER_STREAM_CREATED {
+		err = s.memphisAddStream(s.MemphisGlobalAccountString(), &StreamConfig{
+			Name:         notificationsStreamName,
+			Subjects:     []string{notificationsStreamName},
```

Comment:
make it notificationsStreamName +".>" so that it will wrap all subjects from the pattern notificationsStreamName.something.
This way we will have the option to utilize this stream to more purposes if we will need it in the future